packages/api/src/controllers/room.test.ts

```diff
@@ -34,22 +35,24 @@ describe("controllers/room", () => {
   let adminApiKey: string;
   let nonAdminUser: User;
   let nonAdminToken: string;
+  let roomServiceClient = new RoomServiceClient("");
+  let mockCreateRoom: jest.SpyInstance;
+  let egressClient = new EgressClient("");
```

Comment:
To keep tests independent it might be better to only declare the variables here, but initialize them on `beforeEach`